### ISSUE TITLE
Add to maven central

### ISSUE THREAD

**Issue by KarolisKursevicius** [pokkst/bitcoincashj#13]
Hello. I would like to use this library in my project but it seems that this library is not in maven central. Would it be possible to put this in maven central?

Thank you

**Comment by doc-ef** [comment]
Yeah, I second that request........looks like the answer currently is to pull down sources and build yourself.  FYI, if go this way, the protocol buffer secondary compilation of the .proto files may be needed.

**Comment by QuickMythril** [comment]
have you tried jitpack?

https://jitpack.io/#pokkst/bitcoincashj

**Comment by doc-ef** [comment]
Thanks, jitpack link was just what the doctor ordered ;)

**Comment by doc-ef** [comment]
One more question.  Is there a working BitCoin Cash TestNet faucet.  Have found only one that is still around (https://tbch.googol.cash/) but when enter bchtest:address it gives a 500 error response.  Are there any testnet faucets working for bitcoin cash?
